simulator/src/main/java/org/disaster/routing/DisasterRoutingDashboard.java

```diff
@@ -1,26 +1,33 @@
 package org.disaster.routing;
 
+import org.disaster.routing.analysis.PeopleInSafetyXY;
 import org.disaster.routing.analysis.TripPurposeBy10Min;
+import org.disaster.routing.analysis.TripStatsDisaster;
 import org.matsim.application.analysis.population.TripAnalysis;
 import org.matsim.application.analysis.traffic.TrafficAnalysis;
+import org.matsim.application.analysis.traffic.traveltime.TravelTimeComparison;
 import org.matsim.application.prepare.network.CreateAvroNetwork;
 import org.matsim.simwrapper.Dashboard;
 import org.matsim.simwrapper.Header;
 import org.matsim.simwrapper.Layout;
 import org.matsim.simwrapper.viz.ColorScheme;
 import org.matsim.simwrapper.viz.MapPlot;
 import org.matsim.simwrapper.viz.Plotly;
+import tech.tablesaw.plotly.components.Line;
 import org.matsim.simwrapper.viz.Table;
 import tech.tablesaw.plotly.components.Axis;
 import tech.tablesaw.plotly.traces.BarTrace;
+import tech.tablesaw.plotly.traces.ScatterTrace;
 
 import java.util.List;
 
 public class DisasterRoutingDashboard implements Dashboard {
+    private final String pythonDataPath = "../../../../../../../data/matsim/";
```

Comment:
I don't know why this path works for you. It doesn't work for me on either Windows or MacOS. I'm not sure it it has to be relative to the `analysis` directory, but I didn't have time to investigate further.
(I'm guessing @amalieboegild would experience the same issue if she ran the new dashboard)

![image](https://github.com/user-attachments/assets/c99f2efe-1cd8-43df-ab97-afa6f57611cb)